### ISSUE TITLE
Plugin won't work when another block with display property is present

### ISSUE THREAD

**Issue by wazaraki** [kristerkari/stylelint-declaration-block-no-ignored-properties#7]
Detection of ignored properties stops working as soon as there is another block in the same file which contains a `display` property (with any value, doesn't matter). 

**Version:** 1.0.1

**Test case:**

```css
/* styles.css */
.a {
  display: inline;
  width: 100%; /* <- stylelint should complain about this but doesn't */
}

.b {
  display: xxxx; /* <- this causes the issue */
}
```

**Comment by kristerkari** [comment]
Thanks! I'll look into it and see if I can reproduce that using a test and fix it.

**Comment by kristerkari** [comment]
I wrote a test that is currently failing, to verify that this is indeed a bug:

```js
    {
      code: ".a { display: inline; width: 100px; } .b { display: inline; }",
      message: messages.rejected("width", "display: inline"),
      line: 1,
      column: 22,
      description:
        "display: inline rules out width, height, margin-top and margin-bottom, and float",
    },
``` 
I still need to see how to fix it.

**Comment by kristerkari** [comment]
@wazaraki I added a PR that fix the issue that you reported: https://github.com/kristerkari/stylelint-declaration-block-no-ignored-properties/pull/8

I also added some additional tests to verify that things are working properly. I verified that your code does a actually give a warning after the changes, but does not warn with the old code.

I'll release a new patch version soon.
